### ISSUE TITLE
Getting sent to the backrooms 

### ISSUE THREAD

**Issue by Phrogphone** [DevsterGames/SwordsOfGurrahPublic#21]
Airlink can glitch out on gurrah, which could be blamed on general wifi issues. However sometimes you will be sent to a silent black room with lobby ambience before eventually spawning again, in the lobby, disconnected from whatever server you were in. Consistently, this has caused my quest to physically heat up during the black room sequence, which I fear could damage systems.
1. Go to Gurrah, using airlink.
2. Wait. It happened at random, and there was no real warning outside of mild lag. 

I expected...to play the game, haha. Again, it was entirely random to me.

 no screenshots due to nature of glitch 

Steam game, quest 2 headset

**Comment by benaclejames** [comment]
Are you sure this isn't a case of your pc loosing internet connection?

**Comment by Phrogphone** [comment]
yes because i was still connected to my computer via airlink. no
airlink...no game connection. If that happens, it brings you to the oculus
home right quick
